### ISSUE TITLE
[BUG] `docker-compose` deprecated. Use `docker compose` instead

### ISSUE THREAD

**Issue by brandonkal** [siglens/siglens#1311]
Your install script uses `docker-compose` but it has been replaced by `docker compose`
https://docs.docker.com/compose/migrate/

**Comment by bhorsujal** [comment]
Yes, I faced this issue as well on `Fedora Workstation 41`, which comes with a default `docker compose` (V2) installation with Docker CLI.

Creating a symlink using `sudo ln -s "$(which docker compose)" /usr/local/bin/docker-compose` solved this issue for me.
